package.json

```diff
@@ -31,39 +31,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "9.0.4",
-    "@angular/cdk": "8.0.0",
-    "@angular/common": "^9.0.4",
-    "@angular/compiler": "^9.0.4",
-    "@angular/core": "^9.0.4",
-    "@angular/forms": "^9.0.4",
-    "@angular/material": "8.0.0",
-    "@angular/platform-browser": "^9.0.4",
-    "@angular/platform-browser-dynamic": "^9.0.4",
-    "@angular/router": "^9.0.4",
+    "@angular/animations": "13.0.0",
+    "@angular/cdk": "^13.0.0",
+    "@angular/common": "^13.0.0",
+    "@angular/compiler": "^13.0.0",
+    "@angular/core": "^13.0.0",
+    "@angular/forms": "^13.0.0",
+    "@angular/material": "13.0.0",
+    "@angular/platform-browser": "^13.0.0",
+    "@angular/platform-browser-dynamic": "^13.0.0",
+    "@angular/router": "^13.0.0",
     "core-js": "^2.5.4",
     "rxjs": "^6.5.4",
-    "tslib": "^1.10.0",
-    "zone.js": "~0.10.2"
+    "tslib": "^2.3.1",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.900.4",
-    "@angular-devkit/build-ng-packagr": "~0.900.4",
-    "@angular/cli": "~9.0.4",
-    "@angular/compiler-cli": "^9.0.4",
-    "@angular/language-service": "^9.0.4",
-    "babel-preset-env": "^1.7.0",
+    "@angular-devkit/build-angular": "~13.0.1",
+    "@angular-eslint/builder": "13.0.0-alpha.0",
```

Comment:
We need a todo to update to a stable version of the eslint builders

Reply:
I need to use the alpha version for the schemantics, otherwise there are dependency issues:

![image](https://user-images.githubusercontent.com/32707057/141324794-a791e2db-e7fb-41d5-a015-fc40fe5f5afa.png)